pkg/datapath/linux/node.go

```diff
@@ -1191,6 +1272,26 @@ func (n *linuxNodeHandler) NodeValidateImplementation(nodeToValidate nodeTypes.N
 	return n.nodeUpdate(nil, &nodeToValidate, false)
 }
 
+// NodeNeighDiscoveryEnabled returns whether node neighbor discovery is enabled
+func (n *linuxNodeHandler) NodeNeighDiscoveryEnabled() bool {
+	return n.enableNeighDiscovery
+}
+
+// NodeNeighborRefresh is called to refresh node neighbor table.
+// This is currently triggered by controller neighbor-table-refresh
+func (n *linuxNodeHandler) NodeNeighborRefresh(nodeToRefresh nodeTypes.Node) {
+	n.mutex.Lock()
+	defer n.mutex.Unlock()
+
+	var ifaceName string
+	if option.Config.EnableNodePort {
+		ifaceName = option.Config.DirectRoutingDevice
+	} else {
+		ifaceName = option.Config.EncryptInterface
```

Comment:
I _think_ we need to select the encrypt interface only when IpSec is enabled? \cc @brb